go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
 	github.com/openshift/api v0.0.0-20200901182017-7ac89ba6b971
-	github.com/openshift/machine-api-operator v0.2.1-0.20201002104344-6abfb5440597
+	github.com/openshift/machine-api-operator v0.2.1-0.20201119210302-9643c3accfda
```

Comment:
This is not the right commit.  The commit listed here is: https://github.com/openshift/machine-api-operator/commit/9643c3accfda

We want https://github.com/openshift/machine-api-operator/commit/cb508cd8777d8cfbf3a0dbe475d82ce294a8d56b